nmt.go

```diff
@@ -1,4 +1,5 @@
 // Package nmt contains an NMT implementation.
+// TODO put a link to the specification.
```

Comment:
[question] should this TODO be merged or do we want to create an issue for it?

Reply:
As it was a small todo I figured to make it a todo, but a GH issue also works fine, please check https://github.com/celestiaorg/nmt/issues/98